integration-tests/scripts/install.sh

```
@@ -110,17 +110,23 @@ github_integration() {
-    sed -i "/integrations:/ a \  github:\n\
-    id: \"${GITHUB__APP__ID}\"\n\
-    clientId: \"${GITHUB__APP__CLIENT__ID}\"\n\
-    clientSecret: \"${GITHUB__APP__CLIENT__SECRET}\"\n\
-    host: \"github.com\"\n\
-    publicKey: |-\n\
-    token: \"${GITOPS__GIT_TOKEN}\"\n\
-    webhookSecret: \"${GITHUB__APP__WEBHOOK__SECRET}\"" "$tpl_file"
-    printf "%s\n" "${GITHUB__APP__PRIVATE_KEY}" | sed 's/^/      /' >> "$tmp_file"
-    sed -i "/    publicKey: |-/ r ${tmp_file}" "$tpl_file"
-    rm -rf "$tmp_file"
+    cat << EOF
```

Shellcheck (0.11.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)